README.md

```diff
@@ -50,6 +50,39 @@ For more advanced configuration, please refer to the `curl` documentation.
 
 ## Development
 
+### System Requirements
+Install the following with your system's package manager
+ - opam
+ - make
+ - clang
+ - patch
+ - curl
+ - libcurl-devel
+
+Helper for opam install on linux:
+`bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"`
+
+
+### Setting up OPAM
+```bash
+opam init -y
+eval $(opam env --switch=default)
+```
+
+### OPAM Dependencies
+Install the following with `opam install ...`
+ - ocurl
+ - yojson
+ - containers
+ - base64
+ - ppx_deriving
+ - terminal_size
+ - lwt
+ - lwt_ppx
+ - dune
```

Comment:
These are specified in the `cs_api_client.opam` file. You can achieve the same thing with:

```
opam pin add -k path .
```

Reply:
Thanks Bertrand! I haven't used ocaml before so I was winging it. I'll revise.